srcpkgs/winegui/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'winegui'
+pkgname=winegui
+version=2.3.5
+revision=1
+archs="i686* x86_64*"
+build_style=cmake
+configure_args="-DCUSTOM_PROJECT_VERSION:STRING=${version}"
```

Comment:
As I noted on IRC, you should amend your distribution procress so this argument is unnecessary. Relying on the build process to tell you the version number for your own package is bad practice and, as the upstream author, you are in a position to fix it.

Reply:
I try my best. I don't want a version number checked in the source code. I like the way of dynamically setting the version (based on a git tags or a variable).

However, it seems like GitLab doesn't allow me to have any control of the published source archives. Meaning there is no easy way to prepare those generated source archives. And as I mentioned on IRC I don't like to check in generated code. The version is based on git tags, which work very well when doing package releasing via gitlab ci/cd.

Reply:
Resolved.